### ISSUE TITLE
Move eslint-config-react-tools to devDependencies

### ISSUE THREAD

**Issue by khakulov** [react-tools/react-table#594]
Since new version i am getting a warning:

![image](https://user-images.githubusercontent.com/441436/32322376-a781b018-bfc4-11e7-861d-edc8de537376.png)

**Comment by frankolson** [comment]
I can create A PR for this.

Also looking at the `eslint-config-react-tools`'s dependencies:
```
{
  ...
  "dependencies": {
    "babel-eslint": "^7.2.3",
    "eslint": "^4.3.0",
    "eslint-config-airbnb": "^15.1.0",
    "eslint-plugin-class-property": "^1.0.6",
    "eslint-plugin-import": "^2.7.0",
    "eslint-plugin-jsx-a11y": "^6.0.2",
    "eslint-plugin-react": "^7.4.0"
  },
  "peerDependencies": {
    "eslint-plugin-class-property": "^1.0.6",
    "eslint-plugin-import": "^2.7.0",
    "eslint-plugin-jsx-a11y": "^5.1.1",
    "eslint-plugin-react": "^7.1.0"
  }
}
```

The following need to be added to `devDependencies` as well:
```
eslint-plugin-class-property
eslint-plugin-jsx-a11y
```

And the following can be removed:
```
"eslint": "^4.1.1",
"eslint-config-standard": "^10.2.1",
"eslint-plugin-node": "^5.2.0",
"eslint-plugin-standard": "^3.0.1",
```

**Comment by pjjanak** [comment]
Despite the actual package.json in the repository moving this dependency and the version on npm matching the latest version in the repo, it seems like an outdated package.json was published in npm as this is still an issue.

**Comment by gary-menzel** [comment]
@frankolson There has been a bit of activity in getting the PRs in - I think this has been done now and released in 6.7.5 but it would be good if you could check. Thanks. I've closed it for now.
